rtbkit/plugins/augmentor_interface/http_augmentor_interface.cc

```diff
@@ -0,0 +1,199 @@
+/* http_augmentor_interface.cc
+   Mathieu Stefani, 21 january 2016
+   Copyright (c) 2015 Datacratic.  All rights reserved.
+   
+*/
+
+#include "http_augmentor_interface.h"
+#include "soa/service/http_client.h"
+#include "rtbkit/plugins/bid_request/openrtb_bid_request_parser.h"
+#include "rtbkit/openrtb/openrtb_parsing.h"
+#include "jml/arch/timers.h"
+
+using namespace Datacratic;
+
+namespace RTBKIT {
+
+namespace {
+    DefaultDescription<OpenRTB::BidRequest> desc;
+}
+
+HttpAugmentorInterface::HttpAugmentorInterface(
+    std::shared_ptr<ServiceProxies> proxies,
+    const std::string& serviceName,
+    const Json::Value& config)
+  : AugmentorInterface(std::move(proxies), serviceName)
+{
+    /*
```

Comment:
I'm putting a comment here but let's remove it when we will have a wiki page describing the format